### DISCUSSION TITLE
ANTLR merch

### DISCUSSION THREAD

**Discussion by KvanTTT** [antlr/antlr4#4031] [General]
Hi, @parrt!

Didn't you think about releasing T-shirts or hoodies or other stuff with the ANTLR logo? I'd like to buy it, may be not only me. Let's collect votes under this topic to take a look at number of people who would like as well.

**Comment by parrt** [comment]
Interesting idea! haha. Would the logo really make a good t-shirt? Maybe a subtle small logo over left breast?

"ANTLR Parr-sing since 1989"? haha

**Reply by KvanTTT** [reply]
The picture from ANTLR organization is quite good and suitable:

![image](https://user-images.githubusercontent.com/1150330/208730759-c64a98be-26c5-41ad-a08b-a5ab25486a75.png)

For instance, GitHub has a shop with T-shirts and other stuff: https://www.thegithubshop.com/

**Comment by ericvergnaud** [comment]
We’ll need a picture of you singing !

**Reply by parrt** [reply]
Got a karaoke pic no problem ;)

**Comment by ericvergnaud** [comment]
BTW the LinkedIn group is also a good communication channel, how about we start posting release updates there ?

**Reply by yaccoff** [reply]
I think its a greate idea.

**Reply by parrt** [reply]
sure. I mostly post on Twitter but linkedin is good too.

**Comment by The-Futurist** [comment]
That is a cool logo, gotta say.

**Comment by parrt** [comment]
Certainly is cheap: https://www.customink.com/products/t-shirts/short-sleeve-t-shirts/hanes-perfect-t-crewneck-short-sleeve-t-shirt/176000?min_qty%5B%5D=6&PK=176006

**Reply by The-Futurist** [reply]
> Certainly is cheap: https://www.customink.com/products/t-shirts/short-sleeve-t-shirts/hanes-perfect-t-crewneck-short-sleeve-t-shirt/176000?min_qty%5B%5D=6&PK=176006

Some years ago I did a bunch of baseball caps for a small business I had, they used colored stitching to the precise image file I supplied, really did look superb.

**Comment by parrt** [comment]
It occurs to me that anybody can go try making a T-shirt with the logo. Let me know if you want to try it out and I can send you Adobe illustrator or other vector graphics.

**Comment by parrt** [comment]
Seems like github has a spam problem. I deleted one, but it looks like Eric
must’ve deleted the other that popped up this morning.

On Thu, Dec 29, 2022 at 1:08 AM Haidar Ezio ***@***.***>
wrote:

> Ain't no way this is happening
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/antlr/antlr4/discussions/4031#discussioncomment-4543689>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABLUWPOGR2WJBQLHYS52N3WPVPJRANCNFSM6AAAAAATEQGJWE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Dictation in use. Please excuse homophones, malapropisms, and nonsense.

**Reply by HaidarEzio** [reply]
Yeah, but what's funny is that spam got on Github too, i mean i get it with the other platforms, but for them to come at Github too ? That's just some next level shit xD

Thank you for the info though!
